pkg/gofr/migration/interface.go

```diff
@@ -39,10 +39,10 @@ type Clickhouse interface {
 }
 
 type Cassandra interface {
-	Exec(query string, args ...interface{}) error
-	NewBatch(name string, batchType int) error
-	BatchQuery(name, stmt string, values ...any) error
-	ExecuteBatch(name string) error
+	Exec(ctx context.Context, query string, args ...interface{}) error
+	NewBatch(ctx context.Context, name string, batchType int) error
+	BatchQuery(ctx context.Context, name, stmt string, values ...any) error
+	ExecuteBatch(ctx context.Context, name string) error
```

Comment:
Won't this be a breaking change? Or since, this is on an initial version, we are going ahead with this anyway?

Reply:
Yups it will be but we need to proceed with it in order to support tracing. We will soon release a major version for all datasources